xbmc/ServiceManager.cpp

```diff
@@ -421,6 +422,11 @@ CWinSystemBase &CServiceManager::GetWinSystem()
   return *m_winSystem.get();
 }
 
+CRenderSystemBase *CServiceManager::GetRenderSystem()
```

Comment:
this method is wrong. ServiceManager does not manage RenderSystem, it does not even know that it exists. RenderSystem is owned by WinSystem.

Reply:
Why does ServiceBroker know about RenderSystem then?

Reply:
it was intermediate. clients can call GetWinSystem()->GetRenderSystem()

Reply:
@FernetMenta updated